src/posix/platform/CMakeLists.txt

```diff
@@ -150,6 +150,7 @@ add_library(openthread-posix
     spinel_manager.cpp
     spi_interface.cpp
     system.cpp
+    temp_settings.cpp
```

Comment:
Also add this file in .gn?

Reply:
There is no `.gn` file for the POSIX platform, so no need to add this file to `*.gn`.